pilot/pkg/xds/eds.go

```diff
@@ -65,6 +65,9 @@ func (s *DiscoveryServer) SvcUpdate(shard model.ShardKey, hostname string, names
 func (s *DiscoveryServer) EDSUpdate(shard model.ShardKey, serviceName string, namespace string,
 	istioEndpoints []*model.IstioEndpoint,
 ) {
+	if s.pushStopped.Load() {
```

Comment:
skipping EDS update means the endpointshardz will be stale

Reply:
hmm, imeants to only update cache and not do `configupdate`